dist/index.html

```diff
@@ -157,88 +169,91 @@ <h2 class="card-title">Multi-Post <br> Stories</h3>
                          </div>
                     </div>
                   </div> 
-            </section>
+          </section>
 
-            <section  class="my-skills">
-                <div class="about"> 
-                    <div class="about-self">
-                        <div><h2 class="heading">About  Myself</h2></div>
-                        <div><p class="info">Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p></div>
-                        <h3 href="#" class="connect">LET'S CONNECT</h3>
-                   <ul class="social-media">
-                      <li>
-                        <a href="#" class="icons">
-                          <img src="./img/icons/twitter.svg"/>
-                        </a>
+          <section  class="my-skills">
+            <div class="about about-skills desktop-about-skills"> 
+                <div class="about-self">
+                  <div><h2 class="heading">About  Myself</h2></div>
+                  <div>
+                    <p class="info desk-info">Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+                  </div>
+                  <h3 href="#" class="connect">LET'S CONNECT</h3>
+             <ul class="social-media">
+                <li>
+                  <a href="#" class="icons">
+                    <img src="./img/icons/twitter.svg"/>
+                  </a>
+                </li>
+                <li>
+                  <a href="#" class="icons">
+                    <img src="./img/icons/Linkedin.png"/>
+                  </a>
+                </li>
+                <li>
+                 <a href="#" class="icons">
+                  <img src="./img/icons/micro.svg"/>
+                </a>
+                </li>
+               <li>
+              <a href="#" class="icons"><img src="./img/icons/git.svg"/></a>
+               </li>
+               <li>
+                 <a href="#" class="icons"><img src="./img/icons/hand.svg" /></a>
+               </li>
+             </ul>
```

Comment:
- [ ] please kindly reduce the gap between the about info and the languages so it matches the original design
yours
![Screenshot (1051)_LI](https://user-images.githubusercontent.com/65298048/236390221-aff6339a-47d9-4adb-aed9-96a2c10559e1.jpg)
original
![Screenshot (1052)](https://user-images.githubusercontent.com/65298048/236390327-f7ba41fe-2b45-4520-a8c3-b2a1bdc27a1b.png)